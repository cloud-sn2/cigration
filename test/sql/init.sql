select from master_add_node(:'worker_1_host', :worker_1_port);
select from master_add_node(:'worker_2_host', :worker_2_port);
select from master_add_node(:'worker_3_host', :worker_3_port);

SET citus.enable_ddl_propagation TO off;
create extension if not exists dblink;
create extension if not exists cigration;
RESET citus.enable_ddl_propagation;