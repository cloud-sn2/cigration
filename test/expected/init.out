select from master_add_node(:'worker_1_host', :worker_1_port);
--
(1 row)

select from master_add_node(:'worker_2_host', :worker_2_port);
--
(1 row)

select from master_add_node(:'worker_3_host', :worker_3_port);
--
(1 row)

SET citus.enable_ddl_propagation TO off;
create extension if not exists dblink;
create extension if not exists cigration;
RESET citus.enable_ddl_propagation;
